apiary.apib

```diff
@@ -2158,10 +2161,44 @@ You can search for departments by various search criteria.
                 }
             }
         }
-        
+
+# Group Attributes
+Attributes provide a way to add information to certificates/emails. A full guide on Attributes can be found here: https://help.accredible.com/a-quick-guide-to-attributes
+
+## Attribute Keys [/v1/attribute_keys]
+
+### Create a New Custom Attribute [POST]
```

Comment:
Create a New Custom Attribute **Key** ?

Reply:
I think it would be worth using the full title throughout, to differentiate between the key and value.